algorithm.go

```diff
@@ -5,18 +5,28 @@ import (
 	"crypto/sha256"
 	"errors"
 	"hash"
+	"crypto/hmac"
+	"crypto/rsa"
+	"crypto"
+	"encoding/pem"
+	"crypto/x509"
+	"fmt"
+	"crypto/rand"
 )
 
 var (
-	AlgorithmHmacSha256 = &Algorithm{"hmac-sha256", sha256.New}
-	AlgorithmHmacSha1   = &Algorithm{"hmac-sha1", sha1.New}
+	AlgorithmHmacSha256 = &Algorithm{"hmac-sha256", sha256.New, hmacSign}
```

Comment:
It looks like this could use a goimports run over it https://godoc.org/golang.org/x/tools/cmd/goimports